Libraries/src/Amazon.Lambda.DynamoDBEvents/ExtensionMethods.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using static Amazon.Lambda.DynamoDBEvents.DynamoDBEvent;
+
+namespace Amazon.Lambda.DynamoDBEvents
+{
+    /// <summary>
+    /// Extension methods for working with <see cref="DynamoDBEvent"/>
+    /// </summary>
+    public static class ExtensionMethods
+    {
+        /// <summary>
+        /// Converts a dictionary representing a DynamoDB item to a JSON string.
+        /// This may be useful when casting a DynamoDB Lambda event to the AWS SDK's 
+        /// higher-level document and object persistence classes.
+        /// </summary>
+        /// <remarks></remarks>
+        /// <param name="item">Dictionary representing a DynamoDB item</param>
+        /// <returns>Unformatted JSON string representing the DynamoDB item</returns>
+        public static string ToJson(this Dictionary<string, AttributeValue> item)
+        {
+            return ToJson(item, false);
+        }
+
+        /// <summary>
+        /// Converts a dictionary representing a DynamoDB item to a JSON string.
+        /// This may be useful when casting a DynamoDB Lambda event to the AWS SDK's 
+        /// higher-level document and object persistence classes.
+        /// </summary>
+        /// <remarks></remarks>
+        /// <param name="item">Dictionary representing a DynamoDB item</param>
+        /// <returns>Formatted JSON string representing the DynamoDB item</returns>
+        public static string ToJsonPretty(this Dictionary<string, AttributeValue> item)
+        {
+            return ToJson(item, true);
+        }
+
+        /// <summary>
+        /// Internal entry point for converting a dictionary representing a DynamoDB item to a JSON string.
+        /// </summary>
+        /// <param name="item">Dictionary representing a DynamoDB item</param>
+        /// <param name="prettyPrint">Whether the resulting JSON should be formatted</param>
+        /// <returns>JSON string representing the DynamoDB item</returns>
+        private static string ToJson(Dictionary<string, AttributeValue> item, bool prettyPrint)
+        {
+            if (item == null || item.Count == 0)
+            {
+                return "{}";
+            }
+
+            var stream = new MemoryStream();
```

Comment:
Should we dispose the stream and writer here? I wasn't familiar with `Utf8JsonWriter` and saw that the [Microsoft documentation](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/use-utf8jsonwriter) uses:
```
using var stream = new MemoryStream();
using var writer = new Utf8JsonWriter(stream, options);
```

Reply:
Added `using`s, thanks. Also converted the other two spots to the C# 8 style.